components/ContactInfo.vue

```diff
@@ -3,18 +3,36 @@
 		<h2>
 			Contact Us
 		</h2>
-		<p>
+		<div>
```

Comment:
You might want to use the `[<address>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)` element here instead.